.github/workflows/ci.yml

```diff
@@ -33,6 +65,12 @@ jobs:
         working-directory: ./sdk
     steps:
     - uses: actions/checkout@v2
+    - name: Verify Matrix Version matches Global Version
```

Comment:
If a list of supported Python versions is defined as env variable above together with MIN and MAX versions, probably we do not need this check.

Reply:
The problem is that the CI syntax does not allow for environment variables in the matrix definition. 

I've tried this: 
```
    strategy:
      matrix:
        python-version: [ ${{ env.X_PYTHON_MIN_VERSION }} , ${{ env.X_PYTHON_MAX_VERSION }} ]
```

but sadly, this is not allowed.